doc/release-notes.md

```diff
@@ -2,8 +2,7 @@ Bitcoin Core version 0.12.1 is now available from:
 
   <https://bitcoin.org/bin/bitcoin-core-0.12.1/>
 
-This is a new minor version release, including the BIP9, BIP68 and BIP112
-softfork, various bugfixes and updated translations.
+This is a new major version release, bringing new features and other improvements.
```

Comment:
I don't think this should be changed to major - 0.12.2 will yet again be a minor version release; there will never again be a major release from the 0.12 branch.